src/applications/representative-search/tests/components/SearchResultsHeader.unit.spec.jsx

```diff
@@ -161,13 +172,14 @@ describe('SearchResultsHeader', () => {
           representativeType: 'attorney',
           inProgress: false,
           context: { location: 'new york' },
+          searchArea: '50',
         }}
         pagination={{ totalEntries: 12, currentPage: 1, totalPages: 2 }}
       />,
     );
 
     const expectedString =
-      'Showing 1 - 10 of 12 results for Attorneys within 50 miles of "new york"';
+      'Showing1-10of12resultsfor"Accreditedattorney"within"50miles"of"newyork"';
```

Comment:
Not sure I understand the intention here of stripping out all of the white space. Can you explain?